### ISSUE TITLE
NPG missing provenance

### ISSUE THREAD

**Issue by VladimirAlexiev** [american-art/npg#74]
NPGObjProvenance_2.csv
```
ObjectID,Provenance
29,The artist; purchased 1984 by NPG
35,"Commissioned by NPG 1971, replica of earlier portrait [State Department]; donated 1974 by the sitterԳ law office Covington and Burling, Washington."
```
There is NPGObjProvenance_2-model.pdf, but I checked these two objects and the provenance is missing:

http://data.americanartcollaborative.org/page/npg/object/29
http://data.americanartcollaborative.org/page/npg/object/35

**Comment by workergnome** [comment]
Sadly (particularly for me) I don't think AAC is mapping provenance.  If I'm wrong and ISI is willing to map, I suggest doing it as a typed linguistic object, and can provide a mapping if needed.

**Comment by VladimirAlexiev** [comment]
@bsnikhila Yes, NPGObjProvenance_2-model.pdf maps it as Linguistic Object, but for some reason that mapping is not executed

**Comment by si-npg** [comment]
Inclusion of provenance data is not high priority for NPG
